enterprise/models.py

```diff
@@ -2,42 +2,114 @@
 """
 Database models for enterprise.
 """
-
 from __future__ import absolute_import, unicode_literals
 
-# from django.db import models
+from autoslug import AutoSlugField
+from simple_history.models import HistoricalRecords  # likely a bug in import order checker
+from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
-
+from django.utils.translation import ugettext_lazy as _
 from model_utils.models import TimeStampedModel
 
 
+class EnterpriseCustomerManager(models.Manager):
+    """
+    Model manager for Enterprise Customer model.
+
+    Filters out inactive Enterprise Customers, otherwise works the same as default model manager.
+    """
+
+    # This manager filters out some records, hence according to the Django docs it must not be used
+    # for related field access. Although False is default value, it still makes sense to set it explicitly
+    # https://docs.djangoproject.com/en/1.10/topics/db/managers/#base-managers
+    use_for_related_fields = False
+
+    def get_queryset(self):
+        """
+        Return a new QuerySet object. Filters out inactive Enterprise Customers.
+        """
+        return super(EnterpriseCustomerManager, self).get_queryset().filter(active=True)
+
+
 @python_2_unicode_compatible
 class EnterpriseCustomer(TimeStampedModel):
     """
-    TODO: replace with a brief description of the model.
+    Enterprise Customer is an organization or a group of people that "consumes" courses.
+
+    Members of Enterprise Customer take courses on the edX platform.
```

Comment:
We should probably update this docstring to refer to Users instead of Members, although it may need to be slightly rewritten to make good sense